src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -466,7 +465,6 @@ public void deleteFeedbackSessionsForCourses(List<String> courseIds) {
         return load()
                 .filter("sentPublishedEmail =", false)
                 .filter("isPublishedEmailEnabled =", true)
-                .filter("feedbackSessionType !=", FeedbackSessionType.PRIVATE)
```

Comment:
`datastore-indexes.xml` can be updated.

Reply:
Can you clarify this a little. I didn't understand this. :sweat_smile:

Reply:
This query is supported by a composite index (read more about Datastore indexes [here](https://cloud.google.com/datastore/docs/concepts/indexes)). Since we're updating the query, we should update the [index configuration](https://cloud.google.com/appengine/docs/standard/java/datastore/indexes#index-configuration) too.